scalafmt-dynamic/src/main/scala/org/scalafmt/dynamic/ScalafmtReflect.scala

```diff
@@ -82,18 +75,12 @@ case class ScalafmtReflect(
     }
   }
 
-  private var lastTimestamp = FileTime.fromMillis(0)
-
   def format(filename: Path, code: String): String = {
-    val currentTimestamp = Files.getLastModifiedTime(configFile)
-    if (currentTimestamp.compareTo(lastTimestamp) != 0) {
+    if (config == null) {
```

Comment:
Since `ScalafmtDynamic` create the instance of `ScalafmtReflect`  for each different configuration (sha1 hash of config string) instead of each path of `.scalafmt.conf`, now, we don't need to rely on last modified time for the cache in a instance.